<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="anonymousFindFaces">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Finn ansikter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anonymousOriginal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="glattingOverskrift">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Anonymisering av ansikter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>**Tekst om funksjonalitet**</string>
   </property>
  </widget>
  <widget class="QPushButton" name="liverpool">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Liverpool</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lena">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Lena</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anonymousCode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Vis kode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anonymousAnonymise">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Anonymiser</string>
   </property>
  </widget>
  <widget class="QLabel" name="anonymiseImg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>76</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../hdr-bilder/Adjuster/Adjuster_00032.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>350</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ansikt:</string>
   </property>
  </widget>
  <widget class="QLabel" name="faceCount">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>350</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
